azure/Dockerfile

```diff
@@ -1,6 +1,46 @@
-ARG BENTOML_VERSION=latest
-ARG PYTHON_VERSION=3.7
-FROM bentoml/azure-functions:$BENTOML_VERSION-py$PYTHON_VERSION
+# To enable ssh & remote debugging on app service change the base image to the one below
+# FROM mcr.microsoft.com/azure-functions/python:3.0-python3.8-appservice
+FROM mcr.microsoft.com/azure-functions/python:3.0-python3.8
```

Comment:
We can change the python version base on what's define din bentoml.yml.  Good for follow up PR